config/v3/api_token/send.json

```diff
@@ -33,11 +33,6 @@
       "pattern": "^\\w+$", 
       "description": "The token to remove"
     },
-    "sub_account": {
```

Comment:
i don't think you need to remove from json files as they are auto generated whenever we make changes in backend